### DISCUSSION TITLE
Bogon tag (and detailed address type identification)

### DISCUSSION THREAD

**Discussion by nitefood** [GyulyVGC/sniffnet#208] [Ideas]
Just like for multicast addresses (which you already identify and tag accordingly), I'd like to suggest a _bogon_ tag, along with more detailed address type identification (e.g. RFC1918 private space, or RFC6598 CGN space), to get rid of many unnecessary "unknown location" occurrances.

For example, [these](https://github.com/nitefood/asn/blob/a19482aab9a024c7baa425fdd0b899fbd0bfcb09/asn#L556-L567) are some regex patterns I used for a similar purpose.

**Comment by GyulyVGC** [comment]
That's an amazing suggestion, thank you very much!

**Comment by GyulyVGC** [comment]
#209
